common/frame.go

```diff
@@ -995,9 +996,20 @@ func (f *Frame) InnerHTML(selector string, opts goja.Value) string {
 	}
 
 	applySlowMo(f.ctx)
-	return value.(string)
+
+	if value == nil {
+		return ""
+	}
+
+	val, ok := value.(goja.Value)
+	if !ok {
+		k6Throw(f.ctx, "unexpected innerHTML value type: %T", value)
+	}
+
+	return val.ToString().String()
```

Comment:
Most of these methods could be refactored into helper functions. Notice that `InnerHTML` and `InnerText` are basically identical, not just this part. Actually I just noticed that `InnerText` calls `NewFrameInnerHTMLOptions()` :facepalm: Which is fine in practice since they're identical too, just...

So I'd rather leave this as is, and DRY it in another PR when we decide how to do that.